minicluster/src/main/java/tachyon/master/LocalTachyonCluster.java

```diff
@@ -249,6 +250,17 @@ private void startMaster(TachyonConf testConf) throws IOException {
     // Update the test conf with actual RPC port.
     testConf.set(Constants.MASTER_PORT, String.valueOf(getMasterPort()));
 
+    // If we are using the LocalMiniDFSCluster, we need to update the UNDERFS_ADDRESS to point to
+    // the cluster's current address. This must happen here because the cluster isn't initialized
+    // until mMaster is started.
+    UnderFileSystemCluster ufs = UnderFileSystemCluster.get();
+    // TODO(andrew): Move LocalMiniDFSCluster to the integration-tests project so that we can use
+    // instanceof here instead of comparing classnames.
```

Comment:
we probably don't want to move `LocalTachyonCluster` to `integration-tests`. The reason is `integration-tests` module depend on almost every other module; while `minicluster` module (where `LocalTachyonCluster` sits) is designed to work as mock tachyon for this project as well as other external projects, so we don't want to increase its dependency scope.